### PR TITLE
Qs summary endpage

### PR DESCRIPTION
**Pull request by JagadishPujari**

### CODE REVIEW

File: player/public/coreplugins/org.ekstep.launcher-1.0/renderer/templates/renderer.html [hunk 1]
```diff
@@ -8,6 +8,6 @@
     <!-- <div id="errorDiv" style="z-index: 9999;" ng-show="showMessage" ng-click="showMessage = !showMessage">
     <div ng-bind-html="message" class="bar bar-energized bookshelf_message"></div>
 </div> -->
-    <iframe id="htmlFrame" ng-src="{{currentProjectUrl}}" width="1366" height="768" scrolling="auto" style="display:none;"></iframe>
+    <iframe id="contentPlayer" ng-src="{{currentProjectUrl}}" width="1366" height="768" scrolling="auto" style="display:none;"></iframe>
```
Comment (vinukumar-vs) [inline player/public/coreplugins/org.ekstep.launcher-1.0/renderer/templates/renderer.html]: Dont change this code.
Why do you need this change?

File: player/public/coreplugins/org.sunbird.assess.endpage-1.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "description": "",
+    "publishedDate": "",
+    "renderer": {
+        "main": "renderer/plugin.js",
+        "dependencies": [
+            {"type": "css", "src": "renderer/templates/style.css"}
+        ]
+    },
+    "dependencies": [
+        {"plugin": "org.sunbird.player.userswitcher", "ver": "1.0", "scope": "renderer" }
```
Comment (vinukumar-vs) [inline player/public/coreplugins/org.sunbird.assess.endpage-1.0/manifest.json]: Remove this user-switcher dependency here. Not required for assessment or exam contents

File: player/public/coreplugins/org.sunbird.assess.endpage-1.0/renderer/endpageApp.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        if (startTime) {
+            var totalTime = Math.round((new Date().getTime() - startTime) / 1000);
+            var mm = Math.floor(totalTime / 60);
+            var ss = Math.floor(totalTime % 60);
+            $scope.totalTimeSpent = (mm > 9 ? mm : ("0" + mm)) + ":" + (ss > 9 ? ss : ("0" + ss));
+        } else {
+            $scope.showFeedbackArea = false;
+        }
+    };
+    $scope.openGenie = function(){
```
Comment (vinukumar-vs) [inline player/public/coreplugins/org.sunbird.assess.endpage-1.0/renderer/endpageApp.js]: Remove this code.

File: player/public/coreplugins/org.sunbird.assess.endpage-1.0/renderer/plugin.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+org.ekstep.contentrenderer.baseEndepage.extend({
+    myApp: undefined,
+    templatePath : undefined,
+    controllerPath:undefined,
+    _ngScopeVar: "endPage",
+    _injectTemplateFn: undefined,
+    initialize: function(data) {
+        this.templatePath = EkstepRendererAPI.resolvePluginResource(data.id, data.ver, "renderer/templates/end.html");
+        this.controllerPath = EkstepRendererAPI.resolvePluginResource(data.id, data.ver, "renderer/endpageApp.js");
+        EkstepRendererAPI.addEventListener("renderer:genie:click",this.gotTohome,this)
```
Comment (vinukumar-vs) [inline player/public/coreplugins/org.sunbird.assess.endpage-1.0/renderer/plugin.js]: Remove this unwanted code.

File: player/public/coreplugins/org.sunbird.assess.endpage-1.0/renderer/plugin.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        var instance = this;
+        org.ekstep.service.controller.loadNgModules(this.templatePath, this.controllerPath);
+    },
+    initEndPage:function(event, instance){
+        
+        EkstepRendererAPI.dispatchEvent("renderer:add:template", {templatePath: this.templatePath, scopeVariable: this._ngScopeVar, toElement: '#pluginTemplate'});
+    },
+    gotTohome: function() {
```
Comment (vinukumar-vs) [inline player/public/coreplugins/org.sunbird.assess.endpage-1.0/renderer/plugin.js]: Remove this unwanted code

**Comment by JagadishPujari** [comment]
Closing this PR since it has same with below PR
https://github.com/project-sunbird/sunbird-content-player/pull/271

**Pull request closed without merging**